webpack/assets/javascripts/react_app/common/colors.scss

```diff
@@ -0,0 +1 @@
+$light-grey: #efefef;
```

Comment:
I assume this is from patternfly pallet? https://www.patternfly.org/styles/color-palette/#_